regtests/ww3_tp2.22/input/gx_outf.inp

```diff
@@ -0,0 +1,11 @@
+$ WAVEWATCH III Grid output post-processing (GrADS)
```

Comment:
Are you actually looking at these outputs? If so great! Otherwise, perhaps let's not add this and the *.gs file here.